qa/suites/rados/dashboard/tasks/dashboard.yaml

```diff
@@ -0,0 +1,51 @@
+
+tasks:
+  - install:
+  - ceph:
+      # tests may leave mgrs broken, so don't try and call into them
+      # to invoke e.g. pg dump during teardown.
+      wait-for-scrub: false
+      log-whitelist:
+        - overall HEALTH_
+        - \(MGR_DOWN\)
+        - \(PG_
+        - replacing it with standby
+        - No standby daemons available
+        - \(FS_DEGRADED\)
+        - \(MDS_FAILED\)
+        - \(MDS_DEGRADED\)
+        - \(FS_WITH_FAILED_MDS\)
+        - \(MDS_DAMAGE\)
+        - \(MDS_ALL_DOWN\)
+        - \(MDS_UP_LESS_THAN_MAX\)
+        - \(OSD_DOWN\)
+        - \(OSD_HOST_DOWN\)
+        - \(POOL_APP_NOT_ENABLED\)
+        - pauserd,pausewr flag\(s\) set
+        - Monitor daemon marked osd\.[[:digit:]]+ down, but it is still running
+        - evicting unresponsive client .+
+  - rgw: [client.0]
+  - cephfs_test_runner:
+      fail_on_skip: false
+      modules:
+        - tasks.mgr.test_dashboard
+        - tasks.mgr.dashboard.test_auth
+        - tasks.mgr.dashboard.test_cephfs
+        - tasks.mgr.dashboard.test_cluster_configuration
+        - tasks.mgr.dashboard.test_health
```

Comment:
Hi @smithfarm we deactivated the 'test_health' and 'test_perf_counters' suites because of a know and unresolved issue: https://github.com/ceph/ceph/pull/30864 https://tracker.ceph.com/issues/41538 .
I just want to mention it, we can give it a try anyway if you want to.

Reply:
thanks, @s0nea - updated